code/supervised/Logistic_Regression/logistic_ex1.py

```diff
@@ -0,0 +1,17 @@
+from sklearn.datasets import load_iris

```

Comment:
When i run the code i get this "[0 0]
[[9.81797141e-01 1.82028445e-02 1.44269293e-08]
 [9.71725476e-01 2.82744937e-02 3.01659208e-08]]
0.9733333333333334". 
Then i also get an warning"ConvergenceWarning: lbfgs failed to converge. Increase the number of iterations.
  "of iterations.", ConvergenceWarning)".
Maybe you can just explain what these numbers mean, and if you also get this error maybe explain what it is